properties.schema

```diff
@@ -62,7 +62,7 @@
                   "required": true,
                   "default": "",
                   "title": "Glossary description",
-                  "inputType": "TextArea",
+                  "inputType": "Text",
```

Comment:
This should be left as `TextArea`.

Reply:
I did wonder that too but TBH I think it would be fairly unlikely that someone would to put a load of formatting, links etc here:
![image](https://user-images.githubusercontent.com/1249597/59755789-73eea680-9280-11e9-9185-07473e8d3a11.png)

In fact I seriously doubt anyone changes it from the default...

If we do think it should be `TextArea` we should make the same change in adapt-contrib-resources.